source/includes/connection-snippets/x509-snippets/c-connection.c

```diff
@@ -0,0 +1,37 @@
+// begin x509 connection
+#include <mongoc/mongoc.h>
+
+mongoc_client_t * client;
+mongoc_database_t * database;
+mongoc_collection_t * collection;
+bson_t * command, reply, * insert;
+bson_error_t error;
+char * str;
+bool retval;
+
+mongoc_init();
+
+const mongoc_ssl_opt_t * ssl_default = mongoc_ssl_opt_get_default();
+mongoc_ssl_opt_t ssl_opts = {
+  0
+};
+
+memcpy( & ssl_opts, ssl_default, sizeof ssl_opts);
```

Comment:
Nitpick: I think there are a couple extra spaces here. Should probably be:
```
memcpy(&ssl_opts, ssl_default, sizeof ssl_opts);
```

Reply:
Fixed. Thanks!